builder/index.html

```diff
@@ -162,13 +222,48 @@ <h2>Custom Snippet</h2>
             </calcite-popover-manager>
           </span>
 
-          <calcite-input required type="textarea" id="body"></calcite-input>
+          <calcite-text-area required id="body"></calcite-text-area>
+        </calcite-label>
+        <calcite-label layout="default">
+          <span
+          >Import Snippet<sup></sup>
+            <calcite-popover-manager>
+              <calcite-popover
+                label="Popover label"
+                reference-element="import-snippet-popover"
+              >
+              <div>
+                <p>
+                  <b>Import Snippet</b><br />
+                 Upload JSON snippet files directly from the computer.
+                </p>
+                <p>
+                <p>
+                  <b>Recommendations</b><br />
+                  Ensure that the JSON snippet file adheres to the correct format, including 
+                  required fields such as <b>name</b>, <b>prefix</b>, <b>body</b>, and <b>description</b>. 
+                </p>
+              </div>
+            </calcite-popover>
+            <calcite-icon
+              icon="question"
+              scale="s"
+              class="popover"
+              id="import-snippet-popover"
+            ></calcite-icon>
+          </calcite-popover-manager>
+        </span>
+        
+         <input type="file" id="import-snippet" accept=".json">
```

Comment:
Would you mind using the `<calcite-input type="file" ...>`? ([doc here](https://developers.arcgis.com/calcite-design-system/components/input/))
<img width="1267" alt="Screenshot 2024-10-16 at 16 22 35" src="https://github.com/user-attachments/assets/bbc38757-e372-45f5-b8d1-1e6be7eae302">

For UX purposes, could could add a sample file, just in case; maybe a `sample.json` with something like this:

```json
{
  "A JavaScript only snippet": {
      "scope": "javascript",
      "prefix": "myConsoleLog",
      "body": [
          "console.log(\"Hello Map\");"
      ],
      "description": "Console log \"Hello Map\""
  }
}
```